public/docs/js/latest/guide/making-components.jade

```diff
@@ -16,24 +11,6 @@
       component that uses a <code>&lt;child&gt;</code> component like so:
 
   .code-box
-    pre.prettyprint.linenums.lang-javascript(data-name="es5")
-      code.
-        //ES5
-        function ParentComponent() {
-          this.message = "I'm the parent";
-        }
-        ParentComponent.annotations = [
-          new angular.Component({
-            selector: "parent"
-          }),
-          new angular.View({
-            template:
-              '&lt;h1&gt;{{ message }}&lt;/h1&gt;' +
-              '&lt;child&gt;&lt;/child&gt;',
-            directives: [ChildComponent]
-          })
-        ];
-
     pre.prettyprint.linenums.lang-typescript(data-name="typescript")
       code.
         //TypeScript
```

Comment:
not new in this change, but is the backtick on L21 and L24 correct? Looks like a typo